simtools/model/model_utils.py

```diff
@@ -141,7 +73,7 @@ def is_two_mirror_telescope(telescope_model_name):
     bool
         True if the telescope is a two mirror one.
     """
-    tel_class, tel_type = names.split_telescope_model_name(telescope_model_name)
+    tel_class, tel_type, _ = names.split_telescope_model_name(telescope_model_name)
```

Comment:
The line I refer is this one:
https://github.com/gammasim/simtools/blob/7043d36ca9ecf25f123fa10c0f1e43235c4a2b41/simtools/utils/names.py#L319

But now I see that you are actually building the name in the following lines, so this is fine. It is in general a bit confusing because I don't exactly know what to expect from `_telescope_name`. Would it be ok if we would have a list of allowed names somewhere? We have it separately for subssystem, classes, layout, id and sites, but not for telescope names.

Reply:
There is a full list here: https://github.com/gammasim/simtools/blob/f92fbccbb8d0d4f1d92fef6ccc58db93e2183fce/tests/unit_tests/test_db_handler.py#L424

But I don't really want to copy this list into the docstring, as this might / will change in future.